src/html5/parser/helper.rs

```diff
@@ -90,12 +90,7 @@ impl Html5Parser<'_> {
         }
     }
 
-    pub fn insert_text_helper(
-        &mut self,
-        node: NodeId,
-        position: InsertionPositionMode<NodeId>,
-        token: &Token,
-    ) {
+    pub fn insert_text_helper(&mut self, position: InsertionPositionMode<NodeId>, token: &Token) {
         match position {
             InsertionPositionMode::Sibling {
```

Comment:
in fn appropriate_place_insert also missing node.is_namespace(HTML_NAMESPAC)

Reply:
fixed